test/functional/rpc_net.py

```diff
@@ -290,7 +291,7 @@ def test_addpeeraddress(self):
         by first testing adding a tried table entry before testing adding a new table one.
         """
         self.log.info("Test addpeeraddress")
-        self.restart_node(1, ["-checkaddrman=1"])
+        self.restart_node(1, ["-checkaddrman=1", "-cjdnsreachable"])
```

Comment:
What's the reason for `-cjdnsreachable` here?

Reply:
oops, forgot to revert that. thanks!

Reply:
ohh, got it! thank you!